### ISSUE TITLE
Titles and Tags on List results too long too read and taking whole page.

### ISSUE THREAD

**Issue by ZeroCool940711** [opensemanticsearch/open-semantic-search#281]
Sometimes the titles and tags/annotations/Entities are so long they take the whole screen or you have to scroll down multiples pages, I think there should be an option on the web interface to limit the size or how many characters titles and tags can show and add a `More` button to show the rest just like the description field, tags have this button too but still shows too many tags. I think the titles should not be longer than 1 or 2 lines or 50–71 characters, that's the size Google recommends for titles, the descriptions seem to be ok with the size they have, if I'm right they appear to be 200-850 characters long which takes usually 2-6 lines which is okay, they are not too long or too short, they have the perfect size for that field, the tags/entities below the description should have the same treatment as the Title field, it should be no longer than 2 lines and also they should be clickable, when clicked they should show results for content with those tags/entities just like what happens when you click on a tag that from the right side panel. These things should make the List results more easy to read and more natural to use, because the result page will feel more like Bing and Google which are the most popular search engines.

Here is an example of this happening:
![image](https://user-images.githubusercontent.com/5977640/79742076-22a3af00-82b7-11ea-8838-0a32dcc83ed4.png)

Here another with a longer title:
![image](https://user-images.githubusercontent.com/5977640/79742220-58e12e80-82b7-11ea-8d24-b9b33a3dbc12.png)

**Comment by wAikAp** [comment]
I got the same here? any suggestions?

**Comment by ZeroCool940711** [comment]
@wAikAp nope, havent found a way to limit the title length, in my case noticed it was mostly compressed files the one that had this problem so I removed those files but there are other files with the same problem that I could not fix so, we will have to wait until the developers working on opensemanticsearch decide to fix this, It will probably take a long time for this to be fixed as for what I see most of the issues reported here on the repository are completely ignored by the developers, I think they have paid support through donations and everything reported here on the issue tracker is completely ignored so, try not to have your hopes too high.
